<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolClass</class>
 <widget class="QMainWindow" name="ToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_26">
        <property name="font">
         <font>
          <family>Microsoft JhengHei</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Directional Light</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="Ambient_L_2">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Ambient</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="directional_ambient">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="minimum">
               <double>-200000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Intensity_L_2">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Intensity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="directional_intensity">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="minimum">
               <double>-200000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Specular_L_2">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Specular</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="directional_specular">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="minimum">
               <double>-200000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="Position_L_2">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Rotation</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_19">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="directional_x">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_20">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>Y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="directional_y">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_21">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>Z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="directional_z">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Color_L_2">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_23">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>R</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="directional_r">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_24">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>G</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="directional_g">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_25">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>B</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="directional_b">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteDirectionalLight_Button">
          <property name="text">
           <string>Delete ALL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CreateDirectionalLight_Button">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="PointLight">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Microsoft JhengHei</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Point Light</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="Ambient_L">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Ambient</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="point_ambient">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="minimum">
               <double>-200000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Intensity_L">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Intensity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="point_intensity">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="minimum">
               <double>-200000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Specular_L">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Specular</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="point_specular">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="minimum">
               <double>-200000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Range_L">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Range</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="point_range">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="minimum">
               <double>-200000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="Position_L">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Position</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="point_x">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_10">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>Y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="point_y">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_11">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>Z</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="point_z">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Color_L">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_9">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>R</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="point_r">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_12">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>G</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="point_g">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label_13">
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string>B</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="point_b">
                <property name="minimum">
                 <double>-200000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>200000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="DeletePointLight_Button">
        <property name="text">
         <string>Delete ALL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CreatePointLight_Button">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Position_L_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Position Camera</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_28">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="camera_x">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_29">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="camera_y">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_30">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="camera_z">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="LoadPath">
          <property name="text">
           <string>Path Load XML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LoadButton">
          <property name="text">
           <string>Load XML</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLineEdit" name="SavePath">
          <property name="text">
           <string>Path Save XML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="text">
           <string>Save XML</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Tool.qrc"/>
 </resources>
 <connections/>
</ui>
